healpix_alchemy/constants.py

```diff
@@ -0,0 +1,18 @@
+from astropy.coordinates import ICRS
+from astropy import units as u
+from astropy_healpix import level_to_nside, HEALPix
+from mocpy import IntervalSet
+import sqlalchemy as sa
+
+LEVEL = IntervalSet.HPX_MAX_ORDER
```

Comment:
Let's hope these always stay constant :grimacing:

Reply:
Can you think of a better name for this module, or a better place to put these values and objects? These are not meant to be part of the public API.

Reply:
How about renaming `constants` to `_constants`?